modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -582,6 +582,19 @@ Number of tests run: 2 - Failed: 0 - Errors: 0 - Skipped: 0 - Time elapsed: 2246
 +
 You can now use this connector from your Studio palette.
 +
+
+== Add a Custom Icon to Your Module
```

Comment:
This looks like the same procedure as the other one. If you decide the exact same information needs to be in two places, consider reuse. :-)

Reply:
All of the comments I left on the other procedure apply to this one as well.

Reply:
Yes! Good idea.